[2025-01-12T17:22:15.758+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-12T17:22:15.787+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xgboost_drift_pipeline.check_drift manual__2025-01-12T17:22:13.572963+00:00 [queued]>
[2025-01-12T17:22:15.800+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xgboost_drift_pipeline.check_drift manual__2025-01-12T17:22:13.572963+00:00 [queued]>
[2025-01-12T17:22:15.801+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-12T17:22:15.822+0000] {taskinstance.py:2889} INFO - Executing <Task(BranchPythonOperator): check_drift> on 2025-01-12 17:22:13.572963+00:00
[2025-01-12T17:22:15.851+0000] {standard_task_runner.py:72} INFO - Started process 198 to run task
[2025-01-12T17:22:15.857+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'xgboost_drift_pipeline', 'check_drift', 'manual__2025-01-12T17:22:13.572963+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/train_xgboost_pipeline.py', '--cfg-path', '/tmp/tmpb0ipm7_t']
[2025-01-12T17:22:15.862+0000] {standard_task_runner.py:105} INFO - Job 66: Subtask check_drift
[2025-01-12T17:22:15.891+0000] {logging_mixin.py:190} WARNING - /usr/local/lib/python3.10/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-01-12T17:22:15.934+0000] {task_command.py:467} INFO - Running <TaskInstance: xgboost_drift_pipeline.check_drift manual__2025-01-12T17:22:13.572963+00:00 [running]> on host 42dc6dafdabb
[2025-01-12T17:22:16.032+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='xgboost_drift_pipeline' AIRFLOW_CTX_TASK_ID='check_drift' AIRFLOW_CTX_EXECUTION_DATE='2025-01-12T17:22:13.572963+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-12T17:22:13.572963+00:00'
[2025-01-12T17:22:16.034+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-12T17:22:16.057+0000] {baseoperator.py:421} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-01-12T17:22:16.058+0000] {train_xgboost_pipeline.py:728} INFO - Checking for drift...
[2025-01-12T17:22:16.429+0000] {train_xgboost_pipeline.py:674} INFO - Downloaded data/Ingestion/Financial_with_Sentiment.csv from MinIO to /tmp/Financial_with_Sentiment.csv
[2025-01-12T17:22:16.456+0000] {train_xgboost_pipeline.py:674} INFO - Downloaded data/Ingestion/Financial_with_Sentiment_pseudo.csv from MinIO to /tmp/Financial_with_Sentiment_pseudo.csv
[2025-01-12T17:22:16.920+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:737 DtypeWarning: Columns (2161) have mixed types. Specify dtype option on import or set low_memory=False.
[2025-01-12T17:22:17.461+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:694 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-01-12T17:22:19.885+0000] {logging_mixin.py:190} WARNING - /app/dags/train_xgboost_pipeline.py:694 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2025-01-12T17:22:19.903+0000] {train_xgboost_pipeline.py:748} INFO - Drift detected: True
[2025-01-12T17:22:19.904+0000] {python.py:240} INFO - Done. Returned value was: retrain_model
[2025-01-12T17:22:19.905+0000] {branch.py:38} INFO - Branch into retrain_model
[2025-01-12T17:22:19.906+0000] {skipmixin.py:230} INFO - Following branch ('retrain_model',)
[2025-01-12T17:22:19.929+0000] {skipmixin.py:278} INFO - Skipping tasks [('test_prediction_without_retrain', -1)]
[2025-01-12T17:22:19.986+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-12T17:22:19.987+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=xgboost_drift_pipeline, task_id=check_drift, run_id=manual__2025-01-12T17:22:13.572963+00:00, execution_date=20250112T172213, start_date=20250112T172215, end_date=20250112T172219
[2025-01-12T17:22:20.046+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-12T17:22:20.082+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-12T17:22:20.084+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
